spec/asyncapi.md

```diff
@@ -640,6 +656,40 @@ subscribe:
 ```
 
 
+Using eplicit references to the servers on which the channel is available:
+
+```json
+{
+  "description": "This application publishes WebUICommand messages to this AMQP queue which it had received via WebSockets on another channel",
+  "servers": [
+    { "$ref": "#/components/servers/rabbitmqBrokerInProd" },
+    { "$ref": "#/components/servers/rabbitmqBrokerInStaging" },
+  ],
+  "subscribe": {
+    "message": {
+      "$ref": "#/components/messages/WebUICommand"
+    }
+  },
+  "bindings": {
+    "amqp": {
+      "is": "queue"
+    }
+  }
+}
+```
+
+```yaml
+description: This application publishes WebUICommand messages to this AMQP queue which it had received via WebSockets on another channel
+servers:
+  - $ref: "#/components/servers/rabbitmqBrokerInProd"
+  - $ref: "#/components/servers/rabbitmqBrokerInStaging"
```

Comment:
Why not like this:

```suggestion
  - $ref: "#/servers/amqpBrokerInProd"
  - $ref: "#/servers/amqpBrokerInStaging"
```

Reply:
i understand this is your proposal to linking directly to `#/servers/{name-of-server}`: but `{name-of-server}` (eg, `amqpBrokerInProd`) in this case would be name of a server in the map of server objects (https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#servers-object): if you say this works just as well as pointing to a components object then i'm OK with that.